config/manager/manager.yaml

```diff
@@ -19,19 +19,19 @@ spec:
         - name: CRUNCHY_DEBUG
           value: "true"
         - name: RELATED_IMAGE_POSTGRES_13
-          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-1"
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0"
```

Comment:
🔧 These should stay the same.

Reply:
I thought we had turned some of these back to `-0` for testing because the `-1` images weren't released?

Reply:
Reverted